### ISSUE TITLE
[BUG] Chromium headless doesn't have a default prefers-color-scheme

### ISSUE THREAD

**Issue by mxschmitt** [microsoft/playwright#1499]
**Context:**
- Playwright Version: current tip of tree: c0c9b7f137458dab3224087fab243ecfbf231429
- Operating System: macOS and also reproducible on Linux

**Code Snippet**

Example on Try Playwright: https://try.playwright.tech/?s=6haon

```javascript
const playwright = require("playwright");

(async () => {
  for (const browserType of ['chromium', 'webkit', 'firefox']) {
    const browser = await playwright[browserType].launch();
    const context = await browser.newContext();
    const page = await context.newPage();
    console.log(await page.evaluate(() => matchMedia('(prefers-color-scheme: light)').matches));

    await browser.close();
  }
})();
```

**Describe the bug**

WebKit and Firefox have a default `prefers-color-scheme` but headless Chromium not.

The issue is also persistent in Puppeteer.

**Expected**

```
true
true
true
```

**Actual**

```
false
true
true
```

**Comment by mxschmitt** [comment]
Not sure what the reason for that is, on normal mode without headless it works fine. The issue is also persistent in the Puppeteer code.

**Comment by mxschmitt** [comment]
Update: I found out the solution for that. On my webpage I had no default value set if the `prefers-color-scheme` is not there. That is now fixed: https://github.com/mxschmitt/try-playwright/pull/85/files

But maybe this issue is still relevant, because the other browsers are setting a default `prefers-color-scheme` but headless chromium not.

A different behaviour in the different browser engines would be not that good in my mind.
